### PR TITLE
Fixing minor typo on certificate section

### PR DESCRIPTION
**Pull request by akevinieron**
## What type of PR?
Documentation

## What does this PR do?
Fixing minor typo on certificate section

### Related issue(s)
N/A

### CODE REVIEW

File: docs/compose/setup.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Sets the ``TLS_FLAVOR`` to one of the following
 values:
 
 - ``cert`` is the default and requires certificates to be setup manually;
-- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic ceriticates;
+- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic certicates;
```
Comment (HorayNarea) [inline docs/compose/setup.rst:34]: uhhm… isn't this still a typo?

shouldn't it be:
```suggestion
- ``letsencrypt`` will use the *Letsencrypt!* CA to generate automatic certificates;
```

Comment (Nebukadneza) [reply docs/compose/setup.rst:34]: omfg, now i really feel ashamed :sweat_drops:

Comment (akevinieron) [reply docs/compose/setup.rst:34]: o.o How this happened!! I was totally blind!

**Comment by Nebukadneza** [comment]
bors seems to have hiccups again, not getting notified by mergify …

bors r+

**Comment by Nebukadneza** [comment]
And once more i need to apologize … We didn’t notice the PR was made to merge into the `1.7` branch instead of `master`. Could you change your PR to go onto `master`? Sorry for the late request, and thanks!

**Pull request closed without merging**